streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -486,21 +486,21 @@ <VOut> KStream<K, VOut> flatMapValues(final ValueMapperWithKey<? super K, ? supe
      * Relative order is preserved within each input stream though (i.e., records within one input
      * stream are processed in order).
      *
-     * @param stream
+     * @param otherStream
```

Comment:
Some more side cleanup to align parameter naming across the class (cf `joinXxx()` methods)